src/Raven.Client/Documents/AI/AiOperations.cs

```diff
@@ -64,121 +64,41 @@ public AiOperations ForDatabase(string databaseName)
         return AsyncHelpers.RunSync(() => CreateAgentAsync<TSchema>(configuration));
     }
 
-    /// <summary>
-    /// Asynchronously starts a new chat with an AI agent using a fluent parameter builder.
-    /// </summary>
-    /// <typeparam name="TSchema">The schema type for the chat.</typeparam>
-    /// <param name="identifier">The identifier of the AI agent to chat with.</param>
-    /// <param name="prompt">The initial user prompt.</param>
-    /// <param name="builder">A builder function to define RAG parameters.</param>
-    /// <returns>The result of the chat.</returns>
-    public Task<ChatResult<TSchema>> StartChatAsync<TSchema>(string identifier, string prompt, Func<AiAgentParametersBuilder, AiAgentParametersBuilder> builder, CancellationToken token = default) where TSchema : new()
+    public class AiAgentParametersBuilder
     {
-        var parameters = builder.Invoke(new AiAgentParametersBuilder()).GetParameters();
-        return StartChatAsync<TSchema>(identifier, prompt, parameters, token);
-    }
+        private readonly Dictionary<string, object> _parameters = new();
```

Comment:
remember that the parameters in the configuration are case insensitive

Reply:
I'm pretty sure that those are case _sensitive_, like queries.

See:

```
$Name = 'Fuller'
from "Employees" 
where LastName = $name
```


This will fail.